test/parallel/test-module-isBuiltin.js

```diff
@@ -15,20 +15,28 @@ assert(!isBuiltin('test'));
 assert(!isBuiltin(''));
 assert(!isBuiltin(undefined));
 
+const underscoreModules = [
+  '_http_agent',
+  '_http_client',
+  '_http_common',
+  '_http_incoming',
+  '_http_outgoing',
+  '_http_server',
+  '_stream_duplex',
+  '_stream_passthrough',
+  '_stream_readable',
+  '_stream_transform',
+  '_stream_wrap',
+  '_stream_writable',
+  '_tls_common',
+  '_tls_wrap',
+  'node:_http_agent',
```

Comment:
Is it intentional to test `isBuiltin("node:node:_http_agent")`?

Reply:
nope, sorry I added the `node:_http_agent` just for some manual testing and forgot to remove it :smiling_face_with_tear:, thanks for noticing it :slightly_smiling_face::+1: